src/contracts/evm/APTeamMultiSig/index.ts

```diff
@@ -0,0 +1,53 @@
+import { Interface } from "@ethersproject/abi";
+import { BigNumber } from "@ethersproject/bignumber";
+import { TxLog } from "types/evm";
+import abi from "./abi.json";
+
+const iface = new Interface(abi);
```

Comment:
@misicnenad, `AP`, `REVIEW`, and EndowmentMultisig should have the same interface as they all come from `MultisigGeneric`. we'll separate them once difference on their interfaces appear